hongyeollee/services/userService.js

```diff
@@ -18,6 +20,44 @@ const signup = async (name, password, profileImage, email) => {
   return createUser;
 };
 
+// compare encryption of password and service Token
+const login = async (email, password) => {
+  try {
+    let accessToken = null;
+
+    const hashedPassword = await userDao.getPassword(email);
+    console.log(hashedPassword);
+    const comparePassword = await bcrypt.compare(password, hashedPassword);
+    if (!comparePassword) {
+      const pwErr = new Error("WRONG PASSWORD");
+      throw pwErr;
+    }
+
+    //const time = currentUTCKoreaTime()
+    const activityTime = currentUTCKoreaTime() + 60 * 60 * 24;
+    console.log(activityTime);
+
+    const payload = {
+      iss: "hongyeollee",
+      sub: "test token",
+      iat: currentUTCKoreaTime(),
+      nbf: activityTime,
+    };
+
+    accessToken = jwt.sign(payload, process.env.SECRET_KEY);
+    return accessToken;
+  } catch (err) {
+    console.error(err);
+    throw err;
+  }
+};
+
+const currentUTCKoreaTime = () => {
+  const currentTime = new Date().getTime() / 1000 + 60 * 60 * 9;
+  return Math.floor(currentTime);
+};
```

Comment:
아래와 같이 현재 시간을 구할 때 연산을 하는 상수를 아래와 같이 관리하면 어떨까요?
```suggestion
const DIVISION_TIME_SECOND = 1000 + 60 * 60 * 9;

const currentUTCKoreaTime = () => {
  const currentTime = new Date().getTime() / DIVISION_TIME_SECOND;
  return Math.floor(currentTime);
};
```

Reply:
네 알겠습니다~ 초시간으로 나타낸거에 우리나라 시간을 맞추기 위해서 숫자를 9로 추가하였습니다.
push한 소스코드에는 DIVISION_TIME_SECOND_OF_NINEHOUR로 할당해 주었습니다.